README.md

```diff
@@ -52,5 +52,74 @@ Adapt/adopt/extend the `yml` files from one of your workflow folders.
 
 Implement one failing test and at least one passing test:
 
-- _enter the name of a **passing** test
-- _enter the name of a **failing** test
+##TestCase_01
+To implement minimum check for the function
+    float readingsArray[]={4,4,4,4};
+    int  numOfReadings = sizeof(readingsArray)/sizeof(readingsArray[0]);
+	assert(detectRanges(readingsArray, numOfReadings) == 1);
+
+#Implementation 
+```
+int detectRanges(const float* readingsArray, int numOfReadings)
+{
+	return 1;
+}
+```	
+
+##TestCase_02
+To implement range check for the function when number of ranges is 2 (this also holds good for NULL array)
+    float readingsArray[]={3,3,5,4,10,11,12};
+    int  numOfReadings = sizeof(readingsArray)/sizeof(readingsArray[0]);
+	assert(detectRanges(readingsArray, numOfReadings) == 2);
+
+#Implementation 
+```
```

Comment:
The implementation has suddenly gone from trivial (return 1) to full-fledged. Is it possible to shorten the data to go step by step? It will be needed when you need to detect the range and count the number of readings in each.
Try simple steps - each step shouldn't take more than 15 minutes from implementation to test

Reply:
ya..my approach was to first find the number of ranges possible, then the range is defined and along with count implementation